libs/ledgerjs/packages/cryptoassets/src/types.ts

```diff
@@ -17,3 +17,51 @@ export type ERC20Token = [
   string,
   string
 ];
+
+export type TronToken = [
```

Comment:
Actually you can remove all these, now that they're defined in their own file (also ERC20Token above, please).
Thanks!